extra_decorators.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'base_decorator'
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.upcase
+  end
```

Comment:
- [ ] Kindly make the correct implementation of this method to achieve this [requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_decorator_v2.md#:~:text=Implement%20a%20method%20correct_name%20that%20capitalizes%20the%20output%20of%20%40nameable.correct_name.). You only need to make the first letter capital using the `capitalize` method instead of `upcase` method.